### ISSUE TITLE
Error message about invalid config file does not mention which config file is invalid

### ISSUE THREAD

**Issue by not-an-aardvark** [eslint/eslint#8963]
**Tell us about your environment**

* **ESLint Version:** 4.2.0
* **Node Version:** 8.1.4
* **npm Version:** 5.0.3

**What parser (default, Babel-ESLint, etc.) are you using?**

N/A

**Please show your full configuration:**

(see below)

**What did you do? Please include the actual source code causing the issue.**

My config file looks like this:

```yml
# .eslintrc.yml
extends:
  - ./foo.yml
  - ./bar.yml
```

`foo.yml` looks like this:

```yml
# foo.yml
env: {}
```

`bar.yml`, an invalid config file, looks like this:

```yml
# bar.yml
invalidProperty: 5
```

I have an `index.js` file that looks like this:

```js
a;
```

I tried to lint `index.js`: `node_modules/.bin/eslint index.js`

**What did you expect to happen?**

I expected to get an error about the invalid config file, and I expected the error message to mention that the invalid config file is `bar.yml`.

**What actually happened? Please include the actual, raw output from ESLint.**

A config validation error was thrown, and the following output was produced:

```
[object Object]:
	ESLint configuration is invalid:
	- Unexpected top-level property "invalidProperty".

Referenced from: /path/to/.eslintrc.yml
Error: [object Object]:
	ESLint configuration is invalid:
	- Unexpected top-level property "invalidProperty".

Referenced from: /path/to/.eslintrc.yml
    at validateConfigSchema (/path/to/node_modules/eslint/lib/config/config-validator.js:192:15)
    at Object.validate (/path/to/node_modules/eslint/lib/config/config-validator.js:205:5)
    at loadFromDisk (/path/to/node_modules/eslint/lib/config/config-file.js:549:19)
    at load (/path/to/node_modules/eslint/lib/config/config-file.js:592:20)
    at configExtends.reduceRight (/path/to/node_modules/eslint/lib/config/config-file.js:421:36)
    at Array.reduceRight (native)
    at applyExtends (/path/to/node_modules/eslint/lib/config/config-file.js:405:28)
    at loadFromDisk (/path/to/node_modules/eslint/lib/config/config-file.js:556:22)
    at Object.load (/path/to/node_modules/eslint/lib/config/config-file.js:592:20)
    at Config.getLocalConfigHierarchy (/path/to/node_modules/eslint/lib/config.js:228:44)

```

The message does not mention `bar.yml` (although did does mention that the invalid config was referenced from `.eslintrc.yml`). This is probably because we used to assume that all config-loading errors were related to not being able to find the file, so it didn't make sense to repeat the filename before.

Also, the error message is printed twice. That part is probably a separate issue.
